cwms_radar_api/src/main/java/cwms/radar/api/LevelsController.java

```diff
@@ -47,10 +50,15 @@
 
 public class LevelsController implements CrudHandler {
     private static final Logger logger = Logger.getLogger(LevelsController.class.getName());
+    public static final String EFFECTIVE_DATE = "effective-date";
+    public static final String OFFICE = "office";
+    public static final String DATE = "date";
```

Comment:
good idea. later I'll pull these into a separate class for the other controllers to use.